platform-sdk/swirlds-common/src/testFixtures/java/com/swirlds/common/test/fixtures/merkle/dummy/DummyCustomReconnectRoot.java

```diff
@@ -105,8 +106,9 @@ public void setupWithNoData() {}
      * {@inheritDoc}
      */
     @Override
-    public LearnerTreeView<MerkleNode> buildLearnerView(final ReconnectConfig reconnectConfig) {
-        return new DummyLearnerPushMerkleTreeView(reconnectConfig, this);
+    public LearnerTreeView<MerkleNode> buildLearnerView(
+            final ReconnectConfig reconnectConfig, final ReconnectMapStats mapStats) {
```

Comment:
missing nullity annotation for new param

Reply:
added